graphics.tex

```diff
@@ -132,17 +129,188 @@ \subsection*{Hires}
 \end{tabular}
 \end{center}
 
-As shown above now you can easily convert the bits of each line to its hex value and you finally get 8 bytes of data. This is the central idea in hires graphics and with this concept you save a lot of memory. Here in this example it's 8 Bytes versus 64 Bytes if you have to manage all the color references.
-Let's count it up for a full screen picture: We have 40x25 Blocks, that is 1000 blocks in total. Each block is 8 Bytes or 1 Kilobyte, so we'll result in "only" 10K for a full screen picture.\\
+As shown above now you can easily convert the bits of each line to its hex value and you finally get 8 bytes of data. This is the central idea in hires graphics and with this concept you save a lot of memory. Here in this example it's 8 bytes versus 64 bytes if you have to manage all the color references.
```

Comment:
On first reading, I couldn't grasp what you were trying to convey by the end of the line saying `...if you have to manage all the color references`.

On the 2nd reading, I got your gist, but now thinking we can make the comparison more explicit by saying something like `...versus the prior example where 64 bytes were needed to manage all the colour references.`

Also prefer British `colour` to American `color` in the written text of the manual. I'll admit it might feel a contradiction, when our BASIC has a command called `COLOR`, but ah well, feel like momentum for the documentation has leaned towards British English, and even impakt added a style-guide recently to the repo to encourage its use too.